librz/analysis/block.c

```diff
@@ -889,7 +889,7 @@ static bool automerge_get_predecessors_cb(void *user, const ut64 k, const void *
 
 // Try to find the contiguous predecessors of all given blocks and merge them if possible,
 // i.e. if there are no other blocks that have this block as one of their successors
-RZ_API void rz_analysis_block_automerge(RzList *blocks) {
+RZ_API void rz_analysis_block_automerge(RzList /*<RzAnalysisBlock>*/ *blocks) {
```

Comment:
```suggestion
RZ_API void rz_analysis_block_automerge(RzList /*<RzAnalysisBlock*>*/ *blocks) {
```
isn't it a list of pointers?

Reply:
I see you do that for `char *` but not for more complex types

Reply:
I was using the type of the void in `void *rz_list_iter_t::data`, since every list element is a pointer, but that means `RzList<char*>` would be `RzList<char>` and that doesn't feel right.

I'll change it.

Reply:
yeah, i do agree to define the types of pointers as pointers.